packages/user/src/views/VerifyEmail.tsx

```diff
@@ -0,0 +1,88 @@
+import { useTranslation } from "@dzangolab/react-i18n";
+import { Page } from "@dzangolab/react-ui";
+import { Card } from "primereact/card";
+import { useContext, useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+
+import verifyEmail from "@/supertokens/verifyEmail";
+
+import { UserContextType, useUser, userContext } from "..";
+
+const VerifyEmail = () => {
+  const { t } = useTranslation("user");
+  const [verifyEmailLoading, setVerifyEmailLoading] = useState<boolean>(false);
+  const [status, setStatus] = useState<string | undefined>("");
+  const { user } = useContext(userContext) as UserContextType;
+  const { setUser } = useUser();
```

Comment:
Could we use useUser for both user and setUser? useUser returns the userContext so may be able to remove the context imports.